test/index.js

```diff
@@ -57,8 +57,12 @@ function testCommand(command, expected) {
   var status = 0;
   var stdout;
   var stderr = '';
+  var noWarnings = Number (process.versions.node.split ('.')[0]) >= 9;
   try {
-    stdout = execSync (command, {encoding: 'utf8', stdio: 'pipe'});
+    stdout = execSync (
+      command + (noWarnings ? ' --nodejs --no-warnings' : ''),
```

Comment:
I'm passing `--no-warnings` to the Node executable via the `--nodejs` flag. This is to suppress the warning that Node gives when using `--experimental-modules`.